DNN Platform/Library/Prompt/Command.cs

```diff
@@ -10,7 +10,7 @@ namespace DotNetNuke.Prompt
 {
     [Serializable]
     [JsonObject]
-    public class Command : ICommand
+    public class Command : IDnnCommand
```

Comment:
My testing showed that this change would not be breaking. Since `IDnnCommand` includes `ICommand`, the calling code still has everything it needs (assuming there's not a breaking change within how the interfaces are handled).